Gem/Code/Source/AutoGen/NetworkTestComponent.AutoComponent.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```

Comment:
NetworkTestComponent is not particularly descriptive about what this does.. well, I guess it does test stuff though, but maybe something like randomimpulse component or something might work a bit better?

Reply:
I like that. Renaming.